server/portal/apps/accounts/managers/accounts.py

```diff
@@ -131,7 +96,8 @@ def add_pub_key_to_resource(
         token,
         system_id,
         hostname=None,
-        port=22
+        port=22,
+        pub_key=None
```

Comment:
If we're not saving key pairs anymore, should `pub_key` be a required argument now?

Reply:
Yeah, that makes more sense